ngraph/test/frontend/paddlepaddle/op_fuzzy.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <fstream>
+
+#include "util/engine/test_engines.hpp"
+#include "util/test_control.hpp"
+#include <cnpy.h>
+#include "op_fuzzy.hpp"
+#include "ngraph/ngraph.hpp"
+
+using namespace ngraph;
+using namespace InferenceEngine;
+using namespace ngraph;
+using namespace ngraph::frontend;
+using TestEngine = test::IE_CPU_Engine;
```

Comment:
I believe in unit tests we must not perform inference. Can we just compare with referecnce ngraph function?

CC @ilyachur @zhangYiIntel @nosovmik

Reply:
Hi, this PaddlePaddle frontend is used to convert PaddlePaddle Model to OV Ngraph Model just like ONNX Importer in OV. PaddlePaddle Framework hopes their model can run on OV CPU plugin without accuracy loss. Therefore, we use the IE_CPU_Engine here for test just like ONNX's test cases

Reply:
Accuracy should be covered with e2e tests, not with unit tests for ngraph

Reply:
I am also considering this problem. Some Op, like Deformable Conv, multiclass_nms and matrix_nms, their cpu plugin is WIP. But we can make Op conversion happen once ngraph reference function is implemented. So to use INTERPRETER instead of IE_CPU as test engine, is a good suggestion.

But there is a concern -
Have all Op implemented its reference function correctly? 
This concern comes from a real problem - PDPD yolo-box is decomposed. And tests prove that IE_CPU engine has solid accurate result in comparison to INTERPRETER.

Reply:
@zhangYiIntel @ceciliapeng2011 I agree with @ilya-lavrenov.

We shouldn't duplicate inference tests and mix it with other types of tests.
Please refactor these tests and just compare the graph with reference function.

Inference we can run as a part of e2e tests.

Reply:
Thanks all for your advice. Refactor to Use INTERPRETER_Engine for op Fuzzy test
in #6614